source/guide_gollum.rst

```diff
@@ -0,0 +1,133 @@
+.. author:: Moritz Stückler <https://twitter.com/MoStueck>
+
+.. tag:: lang-ruby
+.. tag:: wiki
+.. tag:: web
+
+.. highlight:: console
+
+######
+Gollum
+######
+
+.. tag_list::
+
+Gollum_ is a simple wiki software written in :manual:`Ruby <lang-ruby>`. It is the software component that drives many popular wiki integrations (e.g. GitHub and GitLab wikis). It is a very simple, small and fast wiki alternative to larger systems like :lab:`Dokuwiki <guide_dokuwiki>`. Gollum is Git-backed, so all changes made through the web interface are also committed to the underlying Git repository.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Ruby <lang-ruby>` and its package manager gem
+  * :manual:`web backends <web-backends>`
+  * :manual:`supervisord <daemons-supervisord>`
+
+
+Prerequisites
+=============
+
+
+Since Ruby and its package manager Gem are already installed by default on Uberspace, one might think, that a simple ``gem install gollum`` would do the trick. Unfortunately that's not the case, because currently (June 2020) Uberspace uses a rather old version of CMake (2.8.12.2). So we need to install a recent version (> 3.5.1) of CMake first, before we can run Gollum.
+
+
+Install recent CMake version
+----------------------------
+
+To install a recent CMake version, the fastest way is to run the official installer script in your home folder (alternatively you could also use brew to install the current CMake version). Make sure to copy the link to the latest version from the `official website <https://cmake.org/download/>`_ (look for "Latest Release" and ``Linux x86_64``).
+
+::
+
+  [isabell@stardust ~]$ bash <(wget -qO- https://github.com/Kitware/CMake/releases/download/v3.17.3/cmake-3.17.3-Linux-x86_64.sh)
```

Comment:
This doesn't seem to work :/ can you try again in a fresh uberspace?

```
Do you accept the license? [yn]: 
y
By default the CMake will be installed in:
  "/home/gollum2/cmake-3.17.3-Linux-x86_64"
Do you want to include the subdirectory cmake-3.17.3-Linux-x86_64?
Saying no will install in: "/home/gollum2" [Yn]: 
y

Using target directory: /home/gollum2/cmake-3.17.3-Linux-x86_64
Extracting, please wait...

[gollum2@belinda ~]$ ll /home/gollum2/cmake-3.17.3-Linux-x86_64
total 0
```

Reply:
Fixed it.